### PR TITLE
rm admin-tools

### PR DESCRIPTION
**Pull request by liuhaoyang**
#### Which issue(s) this PR fixes:

Fixes #

#### Specified Reivewers:
/assgin @your-reviewer

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation usage docs, etc.:

```docs

```

### CODE REVIEW

File: conf/monitor/monitor.yaml [hunk 1]
```diff
@@ -36,9 +36,6 @@ elasticsearch:
 
 admin-tools:
```
Comment (innerpeacez) [inline conf/monitor/monitor.yaml:37]: delete module config

**Review by innerpeacez — CHANGES_REQUESTED** [review]

File: conf/monitor/monitor.yaml (hunk 1 above)
Comment (Effet) [inline conf/monitor/monitor.yaml:37]: remove?

File: conf/collector/collector.yaml [hunk 2]
```diff
@@ -21,6 +21,4 @@ monitor-collector:
         password: "${COLLECTOR_AUTH_PASSWORD:G$9767bP32drYFPWrK4XMLRMTatiM6cU}"
```
Comment (Effet) [inline conf/collector/collector.yaml:21]: sensitive password

**Review by Effet — CHANGES_REQUESTED** [review]

**Review by recallsong — APPROVED** [review]

**Review by innerpeacez — APPROVED** [review]

**Review by Effet — APPROVED** [review]

**Pull request merged**